tests/Faker.Tests/Common/NameTests.cs

```diff
@@ -27,14 +29,30 @@ public void Should_Get_Suffix()
 		}
 
 		[Test]
-		[Repeat(100000)]
+		[Repeat(1)]
 		public void Compute_Time_Elapsed_By_The_First_Method()
 		{
-			Name.First();
+			var sw = new Stopwatch();
+			sw.Start();
+			for (int i = 0; i < 100000; i++)
+			{
+				Name.First();
+			}
+			sw.Stop();
+			Console.WriteLine("Elapsed time without caching {0}msec", sw.ElapsedMilliseconds);
+			sw.Reset();
+
+			sw.Start();
+			for (int i = 0; i < 100000; i++)
+			{
+				Name.FirstCached();
+			}
+			sw.Stop();
+			Console.WriteLine("Elapsed time with caching {0}msec", sw.ElapsedMilliseconds);
```

Comment:
Yes, obviously. This should just help us to run the code and clarify the right way to go. I will remove those unit-test missing the `Assert` stage.